bokehjs/test/integration/regressions.ts

```diff
@@ -881,19 +881,32 @@ describe("Bug", () => {
   })
 
   describe("in issue #10498", () => {
-    it("prevents GridBox from rebuilding when rows or cols properties are modified", async () => {
-      const p1 = fig([300, 300])
-      const p2 = fig([300, 300])
-      p1.scatter({x: [0, 1], y: [0, 1], color: "red"})
-      p2.scatter({x: [1, 0], y: [0, 1], color: "green"})
-      const box = new GridBox({
-        children: [[p1, 0, 0], [p2, 0, 1]],
-        cols: ["300px", "300px"],
-        sizing_mode: "fixed",
-      })
-      const {view} = await display(box, [600, 300])
-      box.cols = ["100px", "500px"]
-      await view.ready
+    const params: Array<"cols" | "rows"> = ["cols", "rows"]
+    params.forEach(uses => {
+      it(
```

Comment:
For a small number of cases, I much more prefer to have a function with an argument and then explicit `it()` invocations for each case, e.g.:

```ts
describe("plot something", () => {
  async function plot(orientation: "rows" | "cols") {
    // TODO
  }
  it("with 'rows'", async () => {
    await plot("rows")
  })
  it("with 'cols'", async () => {
    await plot("cols")
  })
})
Much easier to debug, because you don't need conditional breakpoints.
```

Reply:
This is for future reference. The same applies to `as const`.

Reply:
Oh yes this looks much better, thanks for the suggestion!